scripts/cloud-api-demo/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd ~/develop 
```

Comment:
Add comments regarding the purpose of this script file

Reply:
Added